misc/userscripts/qute-keepassxc

```diff
@@ -351,10 +424,21 @@ def main():
         if not creds:
             error('No credentials found')
             return
-        # TODO: handle multiple matches
-        name, pw = creds[0]['login'], creds[0]['password']
-        if name and pw:
-            qute('jseval -q ' + make_js_code(name, pw))
+        cred = select_account(creds)
+        if not cred:
+            error('No credentials selected')
+            return
+        if args.totp:
+            uuid = cred['uuid']
+            totp = kp.get_totp(uuid)
+            if not totp:
+                error('No TOTP key found')
+                return
+            qute('jseval -q ' + make_js_totp_code(totp))
```

Comment:
This could probably use `:insert-text` or `:fake-key` instead, but seems ok as-is too.